ci/exclude.lst

```diff
@@ -0,0 +1,4 @@
+*venv*
+*.md
+src/backend/functions/setup/*
+src/frontend/*
```

Comment:
Without a build command for the orchestrator, all dev dependencies for the orchestrator will be included in the archive. @svozza thoughts on if this PR should handle that or a separate one? It'd involve creating a build command the orchestrator's `package.json` to do something along the lines of `rm -rf build && rsync -ar . ./build/ --exclude test/ && cd ./build && npm i`, then invoking this as part of the top level build command

Reply:
Yeah, let's do it as a separate PR.

Reply:
for now we should probably exclude all of the `src/*` directory thinking about it. the transcriber is stored in `backend.zip`, and the setup function is inlined. once we add in the orchestrator we can update the exclusion/inclusion lists. will update this

Reply:
updated to only include the template, package.json and README in the published cfn artefact. will add in the orchestrator in a separate PR once it's added to the cfn template

Reply:
Perfect